src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 101;
+    private Random random = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(random.nextInt(MAX_BALL_NUMBER));
+        ball.setColor(randomColor.getRandomColor());
+        return ball;
```

Comment:
Let`s decrease the number of lines in your code. Replace those lines with the next one:
```
return new Ball(random.nextInt(MAX_BALL_NUMBER), randomColor.getRandomColor());
```
But you should add a such`public Ball(String color, int number){}` constructor to the ball class.